packages/xchain-thorchain-query/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@xchainjs/xchain-thorchain-query",
-  "version": "0.5.0",
+  "version": "0.6.0",
```

Comment:
Any reason why we are bumping version by minor and not by patch ?

Reply:
Yeah, there is a new parameter on ThorchainCache constructor, that break retro compatibility for that reason i think it's a minor not a patch

Reply:
happy days then